src/controllers/audience/audienceController.ts

```diff
@@ -340,7 +344,10 @@ export class AudienceController {
 			await this.learningCatalogue
 				.updateAudience(res.locals.course.id, res.locals.audience)
 				.then(() => {
-					res.redirect(`/content-management/courses/${req.params.courseId}/audiences/${req.params.audienceId}/configure`)
+					this.requiredLearningCache.deleteAllIds()
```

Comment:
Need to also clear the learning plan and learning record caches here. Could also look into not awaiting them and having it run in the background